### PR TITLE
Adding ListKeys support for Automation Account

### PR DESCRIPTION
**Pull request by vrdmr**
Adding new method which Lists Automation Keys.

----
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/examples/listAutomationAccountKeys.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "parameters": {
+        "subscriptionId": "subid",
+        "resourceGroupName": "rg",
+        "automationAccountName": "MyAutomationAccount",
+        "typeName": "MyCustomType",
```
[the remark this answers is not in the document]
Comment (vrdmr) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/examples/listAutomationAccountKeys.json]: Thats a mistake. I'll remove it

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/account.json [hunk 2]
```diff
@@ -508,6 +508,55 @@
...
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/listKeys": {
+      "post": {
+        "tags": [
+          "ListKeys"
+        ],
+        "operationId": "Keys_ListByAutomationAccount",
+        "description": "Retrieve the automation keys for the account id.",
```
[the remark this answers is not in the document]
Comment (vrdmr) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/account.json]: Okay.

**Review by subbukambala — APPROVED** [review]

**Comment by sergey-shandar** [comment]
@ravbhatnagar new operations

**Comment by vrdmr** [comment]
@ravbhatnagar Ping

**Comment by vrdmr** [comment]
@ravbhatnagar Ping. Could you please review this PR. Thanks.

**Comment by ravbhatnagar** [comment]
Done

**Review by sergey-shandar — APPROVED** [review]
LGTM

**Pull request merged**